src/_sass/header2.scss

```diff
@@ -0,0 +1,204 @@
+$header2-width:    1080px;
+
+$color-white: #ffffff;
+$color-gray: #586E75;
+$color-gray-light: #EEF1F1;
+
+$font-sansSerif: 'Noto Sans', sans-serif;
+$fontSize-small: 14px;
+
+// --------------------------------------------------
+
+.siteHeader2 {
+  width: 100%;
+  display: flex;
+  margin-right: auto;
+  margin-left: auto;
+  box-sizing: border-box;
+
+  @include mobileAndAbove {
+    max-width: $header2-width;
+    padding-left: $spacing-unit;
+    padding-right: $spacing-unit;
+    padding-top: $spacing-unit * 2;
+    padding-bottom: $spacing-unit * 2;
+  }
+}
+
+.siteHeader2-logoLink {
+  opacity: 1;
+  transition: transform $anim-duration ease-in-out, 
+    opacity $anim-duration-short ease;
+
+  &:hover {
+    background-color: transparent;
+    transform: scale(1.025);
+  }
+  &:active {
+    opacity: 0.8;
+  }
+}
+
+.siteHeader2-nav {
+  @include mobileAndAbove {
+    flex: 1;
+  }
+}
+
+.siteHeader2-navList {
+  list-style-type: none;
+
+  @include mobileAndAbove {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+
+    li + li {
+      margin-left: $spacing-unit * 0.5;
+    }
+  }
+}
+
+@mixin button--siteHeader {
+  font-family: $font-sansSerif;
+  padding-top: $spacing-unit * 0.1;
```

Comment:
i don't think spacing-unit is divisible by 10? we usually do these in powers of 2 (unit / 8, unit / 4, etc)

Reply:
Half pixels do render, but I agree that it's best to keep them integers. Thanks for the note!